Microsoft.Recognizers.Definitions/English/DateTimeDefinitions.cs

```diff
@@ -19,7 +19,7 @@ public static class DateTimeDefinitions
 	{
 		public const string TillRegex = @"(?<till>to|till|until|thru|through|--|-|—|——)";
 		public const string RangeConnectorRegex = @"(?<and>and|through|to|--|-|—|——)";
-		public const string RelativeRegex = @"(?<order>next|upcoming|this|last|past|previous|current|over the)";

```

Comment:
What happened to the "over the" case?

Reply:
We will extract "the"now. "over" is similar as "around", "at" which should not be extracted.